### ISSUE TITLE
ENSO metrics new version released to address conflict with newer numpy version

### ISSUE THREAD

**Issue by lee1043** [CLIVAR-PRP/ENSO_metrics#81]
[conda feedstock](https://github.com/conda-forge/enso_metrics-feedstock) updated needed to enable [conda distribution](https://anaconda.org/conda-forge/enso_metrics).

@acordonez could you please help on this?

**Comment by lee1043** [comment]
@acordonez yes, just new version number for this package. For dependencies, no version regulations were made in [meta.yml](https://github.com/CLIVAR-PRP/ENSO_metrics/blob/master/conda/meta.yaml), which makes this release more critical.

**Comment by acordonez** [comment]
@lee1043 Here's the [pull request](https://github.com/conda-forge/enso_metrics-feedstock/pull/4). The CI is failing for Python 3.12 because of a conflict with installing genutil. Still waiting for the CI to run for the other versions.

**Comment by acordonez** [comment]
@lee1043 Regarding the python version, we don't restrict it in [pcmdi_metrics](https://github.com/conda-forge/pcmdi_metrics-feedstock/blob/main/recipe/meta.yaml#L31) even though pcmdi_metrics will also have failures with higher versions (I think > 3.10) with cdat packages.

**Comment by lee1043** [comment]
@acordonez thank you for reminding me. Then let's keep them consistent.
